.gitignore

```diff
@@ -24,6 +24,7 @@
 .byebug_history
 
 .env
+dump.rdb
```

Comment:
Nice! Also you can config global git to ignore many files. [Check it](https://help.github.com/en/articles/ignoring-files)

Reply:
No, ignorar archivos de forma global es mala idea, nunca sabes si en un proyecto de algo que haces vas a necesitar un dump.rdb porque .rdb es la extension de un lenguaje de programacion que usas en el futuro.

Reply:
:man_shrugging: i use it